src/endpoints.js

```diff
@@ -0,0 +1,19 @@
+const endpoints = {
+  submit: 'http://localhost:3001/api/v1/users',
+  /* POST to API:
+    request object:
+      {
+        firstName: 'string',
+        email: 'string',
+        zipCode: 'num',
+        lang: 'string' ** 'en' or 'es' **
+      }
+    response object:
+    -> WHAT DO WE WANT THE BACKEND TO RETURN UPON A SUCCESSFUL POST?
+      success: "New user successfully created.  A confimation has been sent to ${email}.",
+      status: "200",
+      error: "Unable to create user.  Please provide a valid email."
+  */
+}
+
```

Comment:
What do you think about adding this file to the .gitignore since we know what the response is going to be from the BE, or can always console log to obtain this?

Reply:
I have added further response objects to the endpoints.js file based on the PR that you sent to the backend.  As I stated in an earlier comment, I am pretty attached to my endpoints file, especially come testing time!